src/main.cpp

```diff
@@ -776,7 +776,9 @@ QString UsageArguments ( char** argv )
            "      --ctrlmidich      MIDI controller channel to listen\n"
            "      --clientname      client name (window title and jack client name)\n"
            "\nExample: " +
-           QString ( argv[0] ) + " -s --inifile myinifile.ini\n";
+           QString ( argv[0] ) + " -s --inifile myinifile.ini\n"
+           "\nFor more information and localized help see:\n"
+           "\nhttps://jamulus.io/wiki/Command-Line-Options\n";
```

Comment:
Why the extra blank line?

Reply:
![command-output](https://user-images.githubusercontent.com/13550012/119766331-1b38d700-beb5-11eb-9171-f5a976993fa5.PNG)

Because to me, that made sense....but happy to remove.